src/utils/hooks.ts

```diff
@@ -30,3 +31,26 @@ export const usePurseForBrand = (brand?: Brand<'nat'> | null) => {
 
   return purses?.find(p => p.brand === brand);
 };
+
+export const useAuctionTimer = (schedule: LiquidationSchedule | null) => {
+  const currentTime = useAtomValue(currentTimeAtom);
+
+  const minutesUntilNextAuction =
+    schedule?.nextStartTime &&
+    Math.max(
+      Math.floor((Number(schedule.nextStartTime.absValue) - currentTime) / 60),
+      0,
+    );
+
+  const secondsUntilNextAuction =
+    schedule?.nextStartTime &&
+    Math.max(
+      Math.floor((Number(schedule.nextStartTime.absValue) - currentTime) % 60),
+      0,
+    );
+
+  return minutesUntilNextAuction !== undefined &&
+    secondsUntilNextAuction !== undefined
+    ? `in ${minutesUntilNextAuction}m ${secondsUntilNextAuction}s`
+    : '';
```

Comment:
do you really mean to treat schedule in the past as at zero? I think it might be better to show the negative.

consider,
```suggestion
  const secondsLeft =
    schedule?.nextStartTime &&
    Math.floor(Number(schedule.nextStartTime.absValue) - currentTime);

  if (!secondsLeft) {
    return '';
  }

  return `in ${secondsLeft / 60}m ${secondsLeft % 60}s`;
```

Reply:
Yea it was originally showing negative and it seemed awkward. When it's at 0 it means it's about to disappear next block update after liquidation starts anyway so a very transient state.

I think I'll keep the impl because it takes long to run through the test scenario again and I don't think the suggestion will work without tweaks.